src/apps/datacopy.c

```diff
@@ -655,7 +655,10 @@ transfer_data(const BCPPARAMDATA * params, DBPROCESS * dbsrc, DBPROCESS * dbdest
 	}
 
 	srcdata = (MIGCOLDATA *) calloc(sizeof(MIGCOLDATA), src_numcols);
-
+	if(srcdata == NULL){
+		printf("Could not allocate memory to srcdata\n");
```

Comment:
I would use `stderr` for errors.
Is it not `Could not allocate memory FOR srcdata` ?